modules/pool/pool.gql

```diff
@@ -285,6 +296,145 @@ enum GqlPoolType {
     COW_AMM
 }
 
+type GqlPoolAggregator {
+    """
+    The pool id. This is equal to the address for protocolVersion 3 pools
+    """
+    id: ID!
+    """
+    The chain on which the pool is deployed
+    """
+    chain: GqlChain!
+    """
+    The protocol version on which the pool is deployed, 1, 2 or 3
+    """
+    protocolVersion: Int!
+    """
+    The pool type, such as weighted, stable, etc.
+    """
+    type: GqlPoolType!
+    """
+    The name of the pool as per contract
+    """
+    name: String!
+    """
+    The token symbol of the pool as per contract
+    """
+    symbol: String!
+    """
+    The contract address of the pool.
+    """
+    address: Bytes!
+    """
+    The decimals of the BPT, usually 18
+    """
+    decimals: Int!
+    """
+    The wallet address of the owner of the pool. Pool owners can set certain properties like swapFees or AMP.
+    """
+    owner: Bytes
+    """
+    The factory contract address from which the pool was created.
+    """
+    factory: Bytes
+    """
+    The timestamp the pool was created.
+    """
+    createTime: Int!
+    """
+    The version of the pool type.
+    """
+    version: Int!
+    """
+    Returns all pool tokens, including BPTs and nested pools if there are any. Only one nested level deep.
+    """
+    poolTokens: [GqlPoolTokenDetail!]!
+    """
+    Dynamic data such as token balances, swap fees or volume
+    """
+    dynamicData: GqlPoolDynamicData!
```

Comment:
I suggest we return only what's needed. Currently there is a lot of "UI" stats data that is not needed and is cluttering the type. Same for token lists and nested pools. Skipping some "included" table queries will make it faster.

Reply:
I was just trying to reuse as much as possible as we will refactor the whole thing early next year

Reply:
i had an issue locally where the query stalled when not specifying any chain, let's add a default limit at least, so the query doesn't hang.